ftw/builder/tests/__init__.py

```diff
@@ -14,3 +15,13 @@ def setUp(self):
         self.portal = self.layer['portal']
         setRoles(self.portal, TEST_USER_ID, ['Manager'])
         login(self.portal, TEST_USER_NAME)
+
+
+class FunctionalTestCase(TestCase):
+
+    layer = BUILDER_FUNCTIONAL_TESTING
+
+    def setUp(self):
+        self.portal = self.layer['portal']
+        setRoles(self.portal, TEST_USER_ID, ['Manager'])
```

Comment:
I'd prefer a `grant` method here, e.g.:
```python
    def grant(self, *roles):
        setRoles(self.portal, TEST_USER_ID, list(roles))
        transaction.commit()
```

and then grant the role in each test where it is required, e.g.:

```python
    def test_static_portlet_found_in_leftcolumn(self, browser):
        self.grant('Manager')
        ...
```

The advantage is that we do not do it when it is not necessary and that it is explicitly declared in the test that we have a different role for the test user. It makes reading the test easier. :wink: